src/chunk.h

```diff
@@ -174,4 +174,22 @@ extern TSDLLEXPORT int64 ts_chunk_primary_dimension_end(const Chunk *chunk);
 											 CurrentMemoryContext,                                 \
 											 fail_if_not_found)
 
+static inline bool
+ts_flag_is_set_32(uint32 bitmap, uint32 flags)
+{
```

Comment:
The generic flag manipulation functions should probably live in utils.h since they are not specific to chunk.